.gitignore

```diff
@@ -31,3 +31,4 @@ yarn-error.log*
 *.ntvs*
 *.njsproj
 *.sln
+package-lock.json
```

Comment:
unsure whether this should be in gitignore? I didn't add it, so it must have been added automatically by VS Code or something else

Reply:
Honestly, I do not know whether this makes a difference